config/default.yml

```diff
@@ -98,6 +98,14 @@ Rails/ApplicationRecord:
   VersionAdded: '0.49'
   VersionChanged: '2.5'
 
+Rails/ArelStar:
+  Description: 'Enforces `Arel.star` instead of `"*"` for expanded columns.'
+  Enabled: true
+  Safe: false
+  SafeAutoCorrect: false
```

Comment:
Please remove this due to when `Safe: false` then `SafeAutocorrect` is also `false`.